lighthouse-core/config/default.js

```diff
@@ -121,8 +109,6 @@ module.exports = {
     "dobetterweb/external-anchors-use-rel-noopener",
     "dobetterweb/geolocation-on-start",
     "dobetterweb/link-blocking-first-paint",
-    "dobetterweb/no-console-time",
-    "dobetterweb/no-datenow",
     "dobetterweb/no-document-write",
     "dobetterweb/no-mutation-events",
```

Comment:
probably, we won't be getting the same level of detail now that we aren't listening the entire time. strange that it shows up when there are no entries though...